<?xml version="1.0"?>
<launch>
  <include file="$(find ur_modern_driver)/launch/ur10_bringup.launch">
    <arg name="robot_ip" value="192.38.66.226" />    	    
  </include>

   
  <node name="GR2_control" pkg="rob_auto_cal" type="GR2_control" output="screen" args="/misc/shome/ex09/Thesis/program_RG2.script" />


</launch>